integration-tests/db/db.js

```diff
@@ -107,6 +107,15 @@ const clearAllTables = async () => {
   await deleteRows('report')
 }
 
+const getReportCount = reportStatus => {
+  return db
```

Comment:
ok so now the variable must take 2 parameters but the variable name is singular? 

We don't really want to hardcode the number of parameters and also the query is now doing something different when there is only 1 param passed - its says "it's either this value or null"

To get round this we use pg-format. 
`import format from 'pg-format'` (though const/require as JS)
 
then: 

```
.query({
    text: format(
      ` ...   from report r where r.status in (%L)`,
      statuses
    ),
```
where statuses is an array of status.

Reply:
yes, I was thinking of how to not hardcode the  array. 
Done